test/Base_Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -686,6 +686,34 @@ add_specs suite_builder =
             meta_type = Meta.meta Standard.Base.Network
             meta_type.catch . should_equal (Private_Access.Error Nothing Nothing 'meta' 'No Meta.meta for private objects')
 
+        group_builder.specify "function with argument" <|
```

Comment:
What about test for partially applied function (with unapplied arguments)? Is such a function "resolved"?